PatientWebApplication/HealthClinic.DAL/Service/PatientService.cs

```diff
@@ -15,21 +17,25 @@
 
 namespace HealthClinic.CL.Service
 {
-    public class PatientService 
+    public class PatientService : IPatientService
     {
-        private PatientsRepository patientsRepository { get; set; }
-        
-        public PatientService(MyDbContext context)
+        private IPatientsRepository patientsRepository { get; set; }
+
+        public PatientService(IPatientsRepository ipatientsRepository)
         {
-            patientsRepository = new PatientsRepository(context);
+            patientsRepository = ipatientsRepository;
         }
 
-        public PatientUser Create(PatientUser patientUser)
+        public PatientUser Create(PatientDto patientDto)
         {
+            PatientUser patientUser = PatientAdapter.PatientDtoToPatient(patientDto);
             return patientsRepository.Add(patientUser);
```

Comment:
PatientUser can be passed as a parameter to the 'Add' method from patientsRepository.